radix-engine/src/vm/wasm/prepare.rs

```diff
@@ -1172,9 +1172,12 @@ impl WasmModule {
         // Because the offset can be an `InitExpr` that requires evaluation against an WASM instance,
         // we're using the `wasmi` logic as a shortcut.
         let code = self.module.bytes();
-        WasmiModule::new(&code[..]).map_err(|e| PrepareError::NotInstantiatable {
-            reason: format!("{:?}", e),
-        })?;
+        WasmiModule::new(&code[..])
+            .map_err(|_| PrepareError::NotCompilable)?
```

Comment:
Note from the author.

Was thinking we could extend `PrepareError::NotCompilable` with a `reason` field similarly to `PrepareError::NotInstantiatable`.
I am just unsure if it safe to modify `PrepareError` enum.